assets/src/block-validation/sidebar.js

```diff
@@ -119,21 +132,47 @@ export function Sidebar() {
 				</PanelBody>
 			) }
 
-			{
-				! saved && 0 === validationErrors.length && (
-					<PanelBody opened={ true }>
-						<p>
-							{ __( 'Validation issues will be checked for when the post is saved.', 'amp' ) }
-						</p>
-					</PanelBody>
-				)
-			}
-			{ saved && validationErrors.length === 0 && (
+			{ isFetchingErrors ? (
```

Comment:
The `isFetchingErrors` state isn't seeming to be set properly when first loading a post:


https://user-images.githubusercontent.com/134745/109268471-fe124080-77bf-11eb-9e3e-3b7155268ac2.mov

It does show up when saving a post or when re-validating, however.

Reply:
I'd expect there to be a loading spinner when the post is first opened.

Reply:
That's actually a bug I noticed when working on #5929 and addressed there in https://github.com/ampproject/amp-wp/pull/5929/commits/bba583a08ef662760507f8b6823e756278d4e61b. I'm going to port this fix to this PR.

Reply:
Now addressed in 739d240

Reply:
Perfect.